subtests/docker_cli/run_cgroups/run_cgroups.py

```diff
@@ -55,3 +56,42 @@ class memory_no_cgroup(memory_base):
 
 class memory_negative(memory_base):
     pass
+
+
+class cgroup_parent_invalid_name(cgroup_parent_base):
+    """
+    Invalid value for --cgroup-parent option
+    """
+
+    def initialize(self):
+        super(cgroup_parent_invalid_name, self).initialize()
+        self._setup("/foobar",
+                    "",
+                    "Error response from daemon: Cannot start container {cid}:"
+                    " [8] System error: Invalid slice name /foobar")
```

Comment:
Yet another idea: Just copy paste all the ``self.sub_stuff[blah] = blah`` lines into a separate ``initialize()`` call for each sub-subtest.  I realize it's duplication/repetition, however it's only a few lines, and it would allow one or more sub-subtests to diverge in the future w/o having to add all kinds of extra parameters/logic into the common ``_setup()`` method.  Anyway, something to consider because- if you're going to check/raise ``DockerTestNAError`` on too low a version, doing so inside ``initialize()`` may make more sense than inside ``_setup()`` (name wise).  But I'm not too particular on these things.  Just some ideas.